src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4312,6 +4332,9 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
             }
             ASR::accessType s_access = ASR::accessType::Public;
             ASR::presenceType s_presence = ASR::presenceType::Required;
+        	if (i >= default_arg_index_start){                
```

Comment:
```suggestion
        if (i >= default_arg_index_start){                
```

Reply:
I'll push the new reviewed code now, and I'll work on integration test to push soon.